configs/example/gem5_library/x86-gapbs-benchmarks.py

```diff
@@ -75,29 +75,23 @@
 
 # Following are the list of benchmark programs for gapbs
 
-benchmark_choices = ["cc", "bc", "tc", "pr", "bfs"]
-
-synthetic_choices = ["0", "1"]
-
-size_choices = [
-    "1",
-    "2",
-    "3",
-    "4",
-    "5",
-    "6",
-    "7",
-    "8",
-    "9",
-    "10",
-    "11",
-    "12",
-    "13",
-    "14",
-    "15",
-    "16",
-    "USA-road-d.NY.gr",
-]
+benchmark_choices = {
+    "gapbs-cc-test": "",
+    "gapbs-cc-small": "",
+    "gapbs-cc-medium": "",
+    "gapbs-bc-test": "",
+    "gapbs-bc-small": "",
+    "gapbs-bc-medium": "",
+    "gapbs-tc-test": "",
+    "gapbs-tc-small": "",
+    "gapbs-tc-medium": "",
+    "gapbs-pr-test": "",
+    "gapbs-pr-small": "",
+    "gapbs-pr-medium": "",
+    "gapbs-bfs-test": "",
+    "gapbs-bfs-small": "",
+    "gapbs-bfs-medium": "",
+}
```

Comment:
When this was suggested I had this in mind:

```suggestion
gapbs_suite = obtain_resource("gapbs-benchmark-suite")
benchmark_choices = {resources.get_id() for resource in gapbs_suite}
```

Reply:
Right, this is why #609 is required.

Reply:
This change should still work but if will be very slow as it will download everything even if we do -h.